client/Fireport/Assets/Scripts/AnimationEngine/Animations/DefaultAnimationBehaviour.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+
+namespace AnimationEngine.Animations {
+    public class DefaultAnimationBehaviour : AnimationActionBehaviour {
```

Comment:
This is basically the simplest version of an animation action, because it does nothing.  The main thing you could use this for is to have the animation engine wait in between each action group.